### ISSUE TITLE
the last 60 tests

### ISSUE THREAD

**Issue by huangrt01** [carlclone/CS144#1]
你好，我也在写这个lab，想问一下lab4后面的和linux socket通信的测试你会debug吗？ 我有10个总是过不了，然后拉了一下你的仓库发现好像你的也过不了，不知道发生了什么问题。

**Comment by carlclone** [comment]
是的 , 但是我用  TCPconnection 替换了之后 webget 是可以跑的 , 就没有去管那些了 , 后面我做 lab5 的时候再看看
